tools/batchTestGenerator/internal/config.go

```diff
@@ -1,26 +1,123 @@
 package internal
 
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+)
+
+var defaultPlatformSet = map[string]struct{}{
+	"EKS":                     {},
+	"ECS":                     {},
+	"EC2":                     {},
+	"EKS_ARM64":               {},
+	"EKS_ADOT_OPERATOR":       {},
+	"EKS_ADOT_OPERATOR_ARM64": {},
+	"EKS_FARGATE":             {},
+}
+var allPlatformsSet = map[string]struct{}{"EKS": {},
+	"ECS":                     {},
+	"EC2":                     {},
+	"EKS_ARM64":               {},
+	"EKS_ADOT_OPERATOR":       {},
+	"EKS_ADOT_OPERATOR_ARM64": {},
+	"EKS_FARGATE":             {},
+	"CANARY":                  {},
+	"PERF":                    {},
+	"LOCAL":                   {},
+}
+
+var awsRegionRegex = regexp.MustCompile(`^(us(-gov)?|ap|ca|cn|eu|sa)-(central|(north|south)?(east|west)?)-\d$`)
```

Comment:
```suggestion
var awsRegionRegex = regexp.MustCompile(`^(?:us(?:-gov)?|ap|ca|cn|eu|sa)-(?:central|(?:north|south)?(?:east|west)?)-\d$`)
```

Micro-optimization, but since you're not using the contents of these groups they can be made non-capturing.